src/diepvries/template_sql/effectivity_satellite_dml.sql

```diff
@@ -1,3 +1,17 @@
+-- Calculate minimum timestamp that can be affected by the current load.
+-- This timestamp is used in the MERGE statement to reduce the number of records scanned, ensuring
+-- the usage of the recommended clustering key (r_timestamp :: DATE).
+SET min_timestamp = (
+                      SELECT
```

Comment:
Given CTEs below have a smaller indentation, should we follow the same for this `SET` statement?

Reply:
The SELECT statements is aligned with the parenthesis in both cases.